src/os/bluestore/BlockDevice.cc

```diff
@@ -84,16 +85,22 @@ void IOContext::release_running_aios()
 }
 
 BlockDevice *BlockDevice::create(CephContext* cct, const string& path,
-				 aio_callback_t cb, void *cbpriv, aio_callback_t d_cb, void *d_cbpriv)
+                                 aio_callback_t cb, void *cbpriv,
+                                 aio_callback_t d_cb, void *d_cbpriv)
 {
   string type = "kernel";
   char buf[PATH_MAX + 1];
   int r = ::readlink(path.c_str(), buf, sizeof(buf) - 1);
+
+  dout(1) << __func__ << " path " << path << " type " << type << dendl;
+
   if (r >= 0) {
     buf[r] = '\0';
+#if defined(HAVE_SPDK)
```

Comment:
might want to move this change into a different commit.

Reply:
@tchaikov 
Here plays my git ignorance again.... :-(
How do I split the current 1 commit in 3 parts?
Suggestions

Reply:
i'd suggest
```
git reset HEAD^
git add -p
```